### DISCUSSION TITLE
Welcome to ObjectAid-Fixed Discussions!

### DISCUSSION THREAD

**Discussion by RWPhillips** [RWPhillips/ObjectAid-Fixed#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ttbturcu** [comment]
Hi, I've been trying to get ObjectAid to run in different versions of Eclipse since yesterday. Yesterday I have the problem when I press the finish button at Create PNG file, nothing happened. Today I have the same problem with different versions of Eclipse. That is: 
![Screenshot 2024-09-17 134226](https://github.com/user-attachments/assets/efd8d509-44f4-4072-ab7d-c32525715eb3) 
![image](https://github.com/user-attachments/assets/1584e6af-0308-4664-a27c-d0bd52b674d8)
Can you please help me? Thanks!

**Reply by RWPhillips** [reply]
Personally, I would try the "install different version than originally requested". I did some bit of research and it seems that draw2d may have updated to 3.6.1. I have no idea what the ui error is about. For the draw2d error, you could always unzip the jar file and go change the configuration to update the version of draw2d. I'm not exactly sure how to do that or where it would be done, but moving forward this project will probably need to be edited to work with newer versions. I'm currently finishing up college to go onto university so sadly I don't have the time to mess with it. 

Let me know if you have any luck with it and sorry I cannot be more helpful.

**Reply by RWPhillips** [reply]
Okay, my curiosity got the better of me. I unzipped the jar file and then went into the folder (ObjectAid-1.2.4 xStream1.4.20\plugins). I found the "com.objectaid.uml_1.2.4.jar" file and unzipped that. Inside that folder, I opened the manifest file with Notepad++ and that's where you can edit the requirements for the plugin. Perhaps updating it to 3.6.1 (I would confirm the version of draw2d that you have) could fix at least the first error. This is how I was able to fix the file originally, just by messing around with the files inside the jar until I was able to get it working. Play around with it and let me know if you fix it, I'll gladly update the repository if you can get something working.

![Screenshot 2024-12-08 131715](https://github.com/user-attachments/assets/bfe4e780-46d8-494b-a07e-fe37ce466e1f)
![Screenshot 2024-12-08 131529](https://github.com/user-attachments/assets/4482d89d-d591-40d2-a864-97798e6f5c1b)
